venues/aclweb.org/ACL/migrate_submissions_ARR_ACL.py

```diff
@@ -0,0 +1,362 @@
+                "order": 13,
+                "value-regex": "[\\S\\s]{0,500}",
+                "required": False
+                }
+        }
+    }
+)
+client.post_invitation(commitment)
+
+# Creating SAC groups from Track Name 
+f = open('SAC_groups.csv')
+SAC_name_dictionary = {}
+
+file = csv.reader(f)
+header = []
+rows = []
+header = next(file)
+for row in file:
+    rows.append(row)
+f.close()
+for row in rows:
+    SAC_name_dictionary[row[1]] = row[0]
+
+
+
+# Retrieve all commitment submissions, ACL submissions, and ACL blind submissions  
+notes = openreview.tools.iterget_notes(client,invitation='aclweb.org/ACL/2022/Conference/-/Commitment_Submission')
+submissions = openreview.tools.iterget_notes(client,invitation='aclweb.org/ACL/2022/Conference/-/Submission')
+blind_submissions = {note.forum: note for note in list(openreview.tools.iterget_notes(client, invitation = 'aclweb.org/ACL/2022/Conference/-/Blind_Submission'))}
+
+# Save all submissions in a dictionary by paper_link
+submission_dict = {submission.content['paper_link'].split('=')[1]:submission for submission in submissions}
+
+
+# Iterate through all commitment submissions  
+for note in tqdm(notes):
+    # For each, retrieve old paper from link
+    forum = note.content['paper_link'].split('=')[1]
+    if forum not in submission_dict:
+        original_arr_sub_id = client.get_note(forum).original
+        original_arr_sub = client.get_note(original_arr_sub_id)
+        if original_arr_sub:
+            # Create new note to submit to ACL 
+            acl_sub = openreview.Note(
+                invitation="aclweb.org/ACL/2022/Conference/-/Submission",
+                readers = ["aclweb.org/ACL/2022/Conference/Program_Chairs","aclweb.org/ACL/2022/Conference"], # Should SAC be in readers? 
+                writers = ["aclweb.org/ACL/2022/Conference"],
+                signatures = ["aclweb.org/ACL/2022/Conference"],
+                content = {
+                    "paper_link": note.content["paper_link"],
+                    "paper_type":note.content["paper_type"],
+                    "track":note.content["track"],
+                    "comment":note.content.get("comment"),
+                    "authorids":original_arr_sub.content["authorids"],
+                    "authors": original_arr_sub.content["authors"],
+                    "title":original_arr_sub.content["title"],
+                    "abstract":original_arr_sub.content["abstract"],
+                    "data":original_arr_sub.content.get("data"),
+                    "software":original_arr_sub.content.get("software"),
+                    "pdf":original_arr_sub.content.get("pdf")
+                }  
+            )
+            acl_sub = client.post_note(acl_sub)
+            submission_dict[acl_sub.content['paper_link'].split('=')[1]]=acl_sub
+
+# Iterate through submission dictionary, check if each has a blind submission, and create blind submission if not 
+for forum in submission_dict.keys():
```

Comment:
```suggestion
for forum, acl_submission in submission_dict.items():
```

you can get key and value in the for